src/api/public/apidocs-new/paths/person_login.yaml

```
@@ -1,24 +1,16 @@
-post:
-  summary: Perform changes on a registered person
+put:
+  summary: Update existing person details
   security:
     - basic_authentication: []
   parameters:
     - $ref: '../components/parameters/login.yaml'
-    - in: query
-      name: cmd
-      schema:
-        type: string
-        enum: [change_password,lock,delete]
-      required: true
-      description: "The command to execute against the provided person."
-      example: change_password
   requestBody:
-    description: "The password in plain text when the cmd param is 'change_password'."
-    required: false
+    description: The data to update for the person specified in the login parameter.
```

This sentence is a bit weird. Would this be better?

```suggestion
    description: The new data for the person referred in the login parameter.
```

---

Yes, much better. Thanks!